core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -402,6 +404,25 @@ class KafkaZkClient private (zooKeeperClient: ZooKeeperClient, isSecure: Boolean
     }
   }
 
+  /**
+    * Gets all brokers with broker epoch in the cluster.
+    * @return sequence of brokers in the cluster.
+    */
+  def getAllBrokerAndEpochsInCluster: Seq[(Broker, Long)] = {
+    val brokerIds = getSortedBrokerList
```

Comment:
nits: It seems that intellIj complains here. Can you change it to `getSortedBrokerList()`.

Reply:
Fixed.